src/resonanceReconstruction/breitWigner/singleLevel/Apply/src/call.hpp

```diff
@@ -38,8 +38,8 @@ operator()( const ENDF::ResonanceRange& range,
     throw;
   }
   catch ( ... ) {
-
-    throw std::runtime_error( "The resonance range does not appear to contain "
-                              "SLBW parameters" );
+    Log::error( 
```

Comment:
This is not exactly the best error message. I removed the `runtime_error` because I wanted to see what the original exception is? 

BTW, I found this exception was thrown because `callback` threw an exception. There definitely was SLBW parameters.

Reply:
What was the exception thrown for? There must have been something wrong in the first place.

To be honest, I do not think the use of the logger in a component like resonanceReconstruction is that interesting. I'd prefer to throw proper exceptions and let the logging be handled by the application that uses the component. That's why I used a runtime exception in the first place.

Reply:
Mostly I was trying to address is that the message isn't exactly accurate. The exception that was thrown had nothing to do with whether or not there were SLBW parameters.

Reply:
You didn't change the error message, if that was the problem. You just removed the runtime exception and added a Log message.

As I have written in a previous comment: I'm not a fan of using a Log mechanism in a small component like resonance reconstruction. That's why I used runtime_error in the first place. There's maybe 5 or 10 places where it is used. By including ENDFtk as a dependency, we have "contaminated" the component with the logger.